packages/components/psammead-brand/src/index.jsx

```diff
@@ -66,30 +71,68 @@ const BrandSvg = styled.svg`
   /* stylelint-enable */
 `;
 
-const Brand = ({ brandName, svgHeight, minWidth, maxWidth, svg }) => (
-  <Banner svgHeight={svgHeight}>
+/* eslint-disable react/prop-types */
+const StyledBrand = ({ brandName, svgHeight, svg, maxWidth }) => (
+  <Fragment>
     {svg && (
-      <StyledLink
-        href="https://www.bbc.co.uk/news"
-        maxWidth={maxWidth}
-        minWidth={minWidth}
-      >
+      <Fragment>
         <BrandSvg
           height={svgHeight}
           viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
           xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           aria-hidden="true"
           ratio={svg.ratio}
+          maxWidth={maxWidth}
         >
           {svg.group}
         </BrandSvg>
         <VisuallyHiddenText>{brandName}</VisuallyHiddenText>
-      </StyledLink>
+      </Fragment>
+    )}
+  </Fragment>
+);
+
+/* eslint-disable react/prop-types */
```

Comment:
Also, it can sometimes be ok to disable requiring prop types in storybook, but in prod we should always do it really, even on nested components